main.py

```diff
@@ -102,7 +115,42 @@ def parse_config(config_file):
 def ip_to_int(network):
     """ Takes an IP and returns the unsigned integer encoding of the address """
     return struct.unpack('=L', socket.inet_aton(network))[0]
-    
+
+
+def enrich_event(event):
+    """ Takes the raw event data and enriches by adding process tree metadata """
+    proctree_enriched = []
+    error = ""
+    try:
+        proc = psutil.Process(event.pid)
+        proctree = utils.crawl_process_tree(proc)
+        proctree_enriched = list({"pid": p.pid, "cmdline": " ".join(p.cmdline()), "username":  p.username()} for p in proctree)
+    except Exception as e:
+        error=str(e)
+    return(
+        {"timestamp": datetime.utcnow().isoformat() + 'Z',
+        "pid": event.pid,
+        "proctree": proctree_enriched,
+        "daddr": socket.inet_ntoa(struct.pack('<L', event.daddr)),
```

Comment:
can you include a comment about what `<L` does? sorry if it's obvious, but it would help for people that are new to the codebase or unfamiliar with this low-level struct packing stuff (like me :))

Reply:
Yeah no problem, I have to google it literally every time.

Reply:
Resolved in c9a17c081f0aa5c3c6d78e63cf1e9c42a7110e11